core/domain/caching_services.py

```diff
@@ -173,14 +180,16 @@ def get_multi(namespace, sub_namespace, obj_ids):
     memcache_keys = [
         _get_memcache_key(namespace, sub_namespace, obj_id)
         for obj_id in obj_ids]
-    values = memory_cache_services.get_multi(memcache_keys)
+    values = memory_cache_services.get_multi(memcache_keys) # type: ignore[attr-defined]
     for obj_id, value in python_utils.ZIP(obj_ids, values):
         if value:
-            result_dict[obj_id] = DESERIALIZATION_FUNCTIONS[namespace](value)
+            result_dict[obj_id] = DESERIALIZATION_FUNCTIONS[namespace](value) # type: ignore[operator]
```

Comment:
Why is this ignore needed?

Reply:
as functions are mapped in dict `DESERIALIZATION_FUNCTIONS` https://github.com/oppia/oppia/blob/56a99955260f971a212fe8baf5d16a2d6bc8818b/core/domain/caching_services.py#L83

but when we pass value to these functions, it gives `Cannot call function of unknown type  [operator]` error.
so, to avoid this i added ignore stmt here.

Reply:
I think you need to add a type to the `DESERIALIZATION_FUNCTIONS` and then you can get rid of this ignore.

Reply:
Hey! @vojtechjelinek , i'm bit confused about assigment of `DESERIALIZATION_FUNCTIONS` shall i go with `Dict[str, (Collection| Exploration| Skill| Story| Topic| PlatformParameter| object| object)]`, because in last 2 keys json.loads are the values which retruns python object(general python object) based on json string. can you suggest something ?!!

Reply:
Done!